src/haddock/modules-devel/design/optint/modules/pdb.py

```diff
@@ -0,0 +1,70 @@
+"""PDB related functions."""
+from fccpy import read_pdb
+from fccpy.contacts import get_intermolecular_contacts
+
+from haddock.modules.design.optint.modules.residues import AA_TO_CODE
+
+
+def pdb_to_dict(pdb_f):
+    """Read the PDB into a dictionary."""
+    coord_dic = {}
+    with open(pdb_f) as fh:
+        for line in fh:
+            if line.startswith("ATOM"):
+                chain = line[21]
+                if chain not in coord_dic:
+                    coord_dic[chain] = {}
+                resnum = int(line[22:26])
+                resname_three_letter = line[17:20]
+                if resnum not in coord_dic[chain]:
+                    coord_dic[chain][resnum] = AA_TO_CODE[resname_three_letter]
+    return coord_dic
+
+
+def get_interface_dict(pdb_f, cutoff=5.0):
```

Comment:
does it make sense to re-use (adapt) the `identify_interface` method of the CAPRI class here?

Reply:
It does, but since `identify_interface` is in another module i thought its best to keep the modularity, but we could have a a new `libgeometry` with these functions

Reply:
yep, we could port `identify_interface` to the existing library `libalign`, which does all the geometric stuff